### PR TITLE
fix the Feature Gates status

### PR DESCRIPTION
**Pull request by charrywanganthony**
I checked <https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L25>, update all features status which contains word `v1.12`.

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -38,13 +38,18 @@ different Kubernetes components.
 | `APIListChunking` | `true` | Beta | 1.9 | |
 | `APIResponseCompression` | `false` | Alpha | 1.7 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
-| `AttachVolumeLimit` | `true` | Alpha | 1.11 | |
+| `AttachVolumeLimit` | `true` | Alpha | 1.11 | 1.11 |
+| `AttachVolumeLimit` | `true` | Beta | 1.12 | |
 | `BlockVolume` | `false` | Alpha | 1.9 | |
+| `CPUCFSQuotaPeriod` | `false` | Alpha | 1.8 | 1.11 |
+| `CPUCFSQuotaPeriod` | `false` | Alpha | 1.12 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Why two lines for the same gate?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -38,13 +38,18 @@ different Kubernetes components.
...
 | `CPUManager` | `false` | Alpha | 1.8 | 1.9 |
 | `CPUManager` | `true` | Beta | 1.10 | |
 | `CRIContainerLogRotation` | `false` | Alpha | 1.10 | 1.10 |
 | `CRIContainerLogRotation` | `true` | Beta| 1.11 | |
-| `CSIBlockVolume` | `false` | Alpha | 1.11 | 1.11 |
+| `CSIBlockVolume` | `false` | Alpha | 1.11 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: CSIBlockVolume is still in 1.12?

Comment (charrywanganthony) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: is still alpha

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (charrywanganthony) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: updated.

**Comment by tengqm** [comment]
Thanks for the quick fix.
This change reminds us that there are quite a few new features introduced (alpha) but not documented. Would be good to file some issues for tracking.
/lgtm
/approve

**Pull request merged**